src/features/featurefreetrial.h

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef FEATURE_FREETRIAL_H
+#define FEATURE_FREETRIAL_H
+
+#include "models/feature.h"
+
+constexpr const char* FEATURE_FREETRIAL = "freeTrial";
+
+class FeatureFreeTrial : public Feature {
+ public:
+  FeatureFreeTrial()
+      : Feature(FEATURE_FREETRIAL, "Free trial",
+                false,               // Is Major Feature
+                L18nStrings::Empty,  // Display name
+                L18nStrings::Empty,  // Description
+                L18nStrings::Empty,  // LongDescr
+                "",                  // ImagePath
+                "",                  // IconPath
+                "",                  // link URL
+                "2.9",               // released
+                false,               // Can be enabled in devmode
```

Comment:
If free trials are planned to be released in 2.8.1, should this value reflect that?

Reply:
I wasn’t aware that it’s already for ` 2.8.1` — updated the value!